metaflow/plugins/tagging_cli.py

```diff
@@ -0,0 +1,424 @@
+import click
+import sys
+
+from itertools import chain
+
+from metaflow import namespace
+from metaflow.client import Flow, Run
+from metaflow.util import resolve_identity
+from metaflow.exception import CommandException, MetaflowNotFound
+from metaflow.exception import MetaflowNamespaceMismatch
+
+def format_tags(obj, system_tags, all_tags, by_tag):
+    hdr = ["",
+           "System tags and *user tags*; only *user tags* can be removed",
```

Comment:
We don't have a notion of `user tags` for the end user. Also, `can be removed` should read `can be modified`.

Reply:
How would you phrase the whole sentence?